core/listener_default.go

```diff
@@ -21,13 +21,13 @@ type defaultListener struct {
 var DefalutQuicConfig = &quic.Config{
 	Versions:                       []quic.VersionNumber{quic.Version1, quic.VersionDraft29},
 	MaxIdleTimeout:                 time.Second * 5,
-	KeepAlive:                      true,
+	KeepAlivePeriod:                time.Second * 2,
 	MaxIncomingStreams:             1000,
 	MaxIncomingUniStreams:          1000,
 	HandshakeIdleTimeout:           time.Second * 3,
 	InitialStreamReceiveWindow:     1024 * 1024 * 2,
 	InitialConnectionReceiveWindow: 1024 * 1024 * 2,
-	DisablePathMTUDiscovery:        true,
+	// DisablePathMTUDiscovery:        true,
```

Comment:
Why comment this line?

Reply:
about 20 months ago, we found disable MTU discovery will solve the connection problem between GCP euro datacenter and GCP US datacenter by tcpdump. I checked the code, found connection state will be affected by MTU packet drop then. But in the latest version, I found something here: https://mailarchive.ietf.org/arch/msg/quic/5UEB_J_j633xo9U7gSJ2q1c1NdE/. So I try to enable PMTUD be default, if users find problems, can set this to `true`.

Reply:
Does the QUIC configuration of the client need to be modified?

Reply:
already ups on main branch @venjiang